docs/documentation/using-notify.md

```diff
@@ -0,0 +1,136 @@
+# Using GOV.UK Notify to prototype emails and text messages
+
+You can use GOV.UK Notify to send text messages or emails when users
+interact with your prototype. For example you could send users a
+confirmation email sent at the end of a journey.
+
+## Sign up for a GOV.UK Notify account
+
+If you have a government email address you can sign up for an account at
+https://www.gov.uk/notify
+
+You need an account before you can use GOV.UK Notify to send text
+messages or emails.
+
+## Getting an API key
+
+An API key is like a password that lets your prototype talk to Notify.
+Your prototype sends this key every time it asks Notify to do something.
+
+To get a key:
+- sign into GOV.UK Notify
+- go to the ‘API integration’ page
+- click ‘API keys’
+- click the ‘Create an API’ button
+- choose the ‘Team and whitelist’ option
+- copy the key to your clipboard
+
+### Saving your key on your computer
+
+This will let your prototype talk to Notify while it’s running on your
+computer.
+
+To save the key on your computer, run this command in your Terminal, in
+the root folder of your prototype (where xxxxxxx is a key you’ve copied
+from Notify):
```

Comment:
I think it might be less daunting to just tell people to add this line to end of `.env` - and they'll use whatever tool is familiar to them. The only downside is there's a possibility `.env` doesn't exist (before running for the first time), though if so people just need to run the kit.

Reply:
a17b6b3